packages/rpm/polymc.spec

```diff
@@ -1,13 +1,74 @@
-%global _origdir %(pwd)
 
-Name: polymc
-Version: 
-Release: 1%{?dist}
-Summary: A custom launcher for Minecraft
-License: GPLv3
-URL: https://polymc.org/
+%global libnbtplusplus_commit       dc72a20b7efd304d12af2025223fad07b4b78464
+%global libnbtplusplus_shortcommit  %(c=%{libnbtplusplus_commit}; echo ${c:0:7})
+%global quazip_commit               c9ef32de19bceb58d236f5c22382698deaec69fd
+%global quazip_shortcommit          %(c=%{quazip_commit}; echo ${c:0:7})
+
+Name:           polymc
+Version:        1.0.4
+Release:        2%{?dist}
+Summary:        Minecraft launcher with ability to manage multiple instances
+
+#
+# CC-BY-SA
+# ---------------------------------------
+# launcher/resources/multimc/
+#
+# BSD 3-clause "New" or "Revised" License
+# ---------------------------------------
+# application/
+# libraries/LocalPeer/
+# libraries/ganalytics/
+#
+# Boost Software License (v1.0)
+# ---------------------------------------
+# cmake/
+#
+# Expat License
+# ---------------------------------------
+# libraries/systeminfo/
+#
+# GNU Lesser General Public License (v2 or later)
+# ---------------------------------------
+# libraries/rainbow
+#
+# GNU Lesser General Public License (v2.1 or later)
+# ---------------------------------------
+# libraries/iconfix/
+# libraries/quazip/
+#
+# GNU Lesser General Public License (v3 or later)
+# ---------------------------------------
+# libraries/libnbtplusplus/
+#
+# GPL (v2)
+# ---------------------------------------
+# libraries/pack200/
+#
+# ISC License
+# ---------------------------------------
+# libraries/hoedown/
+#
+# zlib/libpng license
+# ---------------------------------------
+# libraries/quazip/quazip/unzip.h
+# libraries/quazip/quazip/zip.h
+#
+
+License:        CC-BY-SA and ASL 2.0 and BSD and Boost and LGPLv2 and LGPLv2+ and LGPLv3+ and GPLv2 and GPLv2+ and GPLv3 and ISC and zlib
```

Comment:
full license constructed from all vendored dependenices and verified using license-validate

```
$ license-validate -v 'CC-BY-SA and ASL 2.0 and BSD and Boost and LGPLv2 and LGPLv2+ and LGPLv3+ and GPLv2 and GPLv2+ and GPLv3 and ISC and zlib'
Approved license
```